charts/astronomer/values.yaml

```diff
@@ -15,19 +15,19 @@ tolerations: []
 images:
   commander:
     repository: astronomerinc/ap-commander
-    tag: 0.18.0
+    tag: 0.21.0
     pullPolicy: IfNotPresent
   registry:
     repository: astronomerinc/ap-registry
     tag: 3.11.5-4
     pullPolicy: IfNotPresent
   houston:
-    repository: astronomerinc/ap-houston-api
-    tag: 0.20.9
+    repository: asoldatenko/ap-houston-api
+    tag: 0.21.3
     pullPolicy: IfNotPresent
   astroUI:
     repository: astronomerinc/ap-astro-ui
-    tag: 0.20.2
+    tag: 0.21.0
```

Comment:
based on https://app.circleci.com/pipelines/github/astronomer/astro-ui/1878/workflows/6bcad779-eb5a-4ed3-9d2c-7f38a369e31e/jobs/4191

Reply:
I think @samblackk is creating a new release for this, so it will be `0.21.1`.

Reply:
done